### PR TITLE
Add support for Nunjucks using PrettyDiff

### PR DESCRIPTION
**Pull request by rtablada**
Looking at the PrettyDiff beautifier, it seems that the support for nunjucks templates already exists.
But, this PR adds the required config to hook up atom-beatuify to recognize the Nunjucks grammar.

### CODE REVIEW

File: src/beautifiers/prettydiff.coffee [hunk 1]
```diff
@@ -48,6 +48,7 @@ module.exports = class PrettyDiff extends Beautifier
     EJS: true
     HTML: true
     Handlebars: true
+    Nunjucks: true
```
Comment (Glavin001) [inline src/beautifiers/prettydiff.coffee]: Pretty Diff is using `auto` by default, which will likely work. However, there should also be a `case` below for handling `Nunjucks` and mapping to the appropriate Pretty Diff language, such as `html`.

**Comment by prettydiff** [comment]
I just looked at Nunjucks.  The vocabulary and syntax appears to be identical to Twig HTML templates, which Pretty Diff supports.

**Comment by Glavin001** [comment]
Thanks, @prettydiff !

**Comment by Glavin001** [comment]
Published to v0.29.12

**Comment by Glavin001** [comment]
Thank you @rtablada for contributing!

**Pull request merged**